Functions.Templates/Templates/EventHubTrigger-FSharp/metadata.json

```diff
@@ -1,13 +1,14 @@
 {
-    "defaultFunctionName": "EventHubTriggerFSharp",
-    "description": "$EventHubTriggerFSharp_description",
-    "name": "EventHubTrigger",
-    "language": "F#",
-    "category": [
-        "$temp_category_core",
-        "$temp_category_dataProcessing"
-    ],
-    "enabledInTryMode": false,
+  "defaultFunctionName": "EventHubTriggerFSharp",
```

Comment:
also fixing spacing here, it looks like

Reply:
I'm not seeing anything wrong with the spacing...?

Reply:
Just commenting on why the lines were touched. I was confused at first glance.